fastestimator/pipeline.py

```diff
@@ -62,11 +63,12 @@ def _verify_inputs(self, **kwargs):
     def _verify_dataset(self, mode: str, dataset: DataSource, **kwargs) -> bool:
         if isinstance(dataset, Dataset):
             # batch_size check
-            assert isinstance(self.batch_size, (Scheduler, int)), \
+            assert isinstance(self.batch_size, (Scheduler, int)) or self.batch_size is None, \
```

Comment:
assert isinstance(self.batch_size, (Scheduler, int, type(None)))

saves on the extra or check

Reply:
nice, done